crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -61,14 +59,14 @@ pub struct DataTransfer {
 impl DataTransfer {
     /// Create a new data transfer instance and initialize it with the data already in
     /// `local_data_folder_path`. If `local_data_folder_path` or a path within
-    /// `local_data_folder_path` could not be read, return [`DeltaTableError`].
+    /// `local_data_folder_path` could not be read, return [`crate::error::ModelarDbServerError`].
```

Comment:
Maybe "[\`ModelarDbServerError\`]\(crate::error::ModelarDbServerError\)" instead. If it is changed here, also change it elsewhere in the file.

Reply:
Fixed.